docs/sql/refresh.txt

```diff
@@ -54,6 +54,13 @@ table is completed.
     REFRESH OK (... sec)
 
 
+.. note::
+
+    If one of the tables in the statement doesn't exists in the cluster none of 
+    them get refreshed. However, an exception is returned if a table is not available 
+    or unknown.
+
```

Comment:
If one or more tables or partitions do not exist, none of the given tables/partitions are refreshed and an error is returned. The error returns only the first non-existent table/partition.